securedrop/journalist_app/api.py

```diff
@@ -309,6 +314,57 @@ def get_all_replies():
         return jsonify(
             {'replies': [reply.to_json() for reply in replies if reply.source]}), 200
 
+    @api.route('/seen', methods=['GET', 'POST'])
+    @token_required
+    def seen():
+        """
+        Lists or marks the source conversation items that the journalist has seen.
+        """
+        user = get_user_object(request)
+
+        if request.method == 'GET':
```

Comment:
We discussed in [the spec](https://docs.google.com/document/d/1szPEKI1gTpRiM_YI87plKgiW39T7zT4Y4HG0uFnsJD4)two possible implementations for this so was wondering why you went in this direction (perhaps for performance reasons as described in https://github.com/freedomofpress/securedrop/issues/5475?)

Reply:
Both are implemented here (though I just realized `/seen`'s `GET` is incomplete, as it's only returning the authenticated client's seen items -- will fix when I rebase to catch up to the latest version of #5505).

Reply:
okay, I see that you added seen records in the /replies and /submissions endpoints :+1:. why did you also add a /seen endpoint to get all seen records for the currently logged in journalist. We never discussed doing this since it's not in scope for the seen/unseen feature - we did discuss providing **all** seen records in a new endpoint as an alternative option to updating the /replies and /submissions endpoint. or am i misunderstanding your change? it could be useful to start a docs draft pr similar to https://github.com/freedomofpress/securedrop-docs/pull/3 to provide clarity for reviewers.

Reply:
In the comment immediately above this, I mentioned that it was incorrect at that point. It has since been changed to match the spec, returning all the records. If you're reviewing locally, make sure to pull frequently. Or just wait; I should be moving this from draft to ready for review shortly.

Reply:
oops sorry about that. will wait!